docs/js/i18n.md

```diff
@@ -84,7 +84,17 @@ You can extract all i18n language within each component by running the following
   npm run extractIntl
 ```
 
-This will extract all language into i18n JSON files in `app/translations`. This allows you to easily add and maintain multi-language support for your application. You can add additional languages by adding an empty generation slot in the `internals/scripts/extractIntl` script and then adding the additional language JSON in `app/i18n.js`.
+This will extract all language into i18n JSON files in `app/translations`.
+
+## Adding A Language
+
+You can add a language by running the generate command:
+
+```
+  npm run generate
```

Comment:
Let's change this to `npm run generate language` (plop supports that :wink: ) and remove the "The select…" sentence part below.